advisories/a71655.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Technical Advisory 71655
+summary: Zigzag joins could potentially produce incorrect results
+toc: true
+---
+
+Publication date: Nov 2, 2021
+
+## Description
+
+Zigzag joins, which may be performed when two or more columns are held constant in a query filter on a table with two or more indexes with those columns, could potentially produce incorrect results.
+
+Incorrect results occur only when the indexes in question have multiple columns, the leading columns are held constant by the filter, and the trailing columns are the same amongst the indexes and are nullable. Incorrect results occur only when these nullable columns have null values.
+
+CockroachDB versions 19.2, 20.1, 20.2, and 21.1 are affected by this bug until maintenance versions XXXXX and XXXXXX. Versions 19.2 and 20.1 are no longer eligible for [maintenance support](../releases/release-support-policy.html), so there will be no patches for these versions.
+
+## Statement
+
+This bug was resolved in CockroachDB by PR [71758](https://github.com/cockroachdb/cockroach/pull/71758), which corrects the bug by no longer planning zigzag joins when they produce incorrect results.
+
+The fix has been applied to maintenance versions XXXX and XXXX of CockroachDB.
```

Comment:
Can we temporarily edit this to say something like "The fix will be included in upcoming patches to 20.2 and 21.1. Please see mitigation steps below to avoid the issue until patches are released.".